pkgs/applications/networking/instant-messengers/mirage/default.nix

```
@@ -1,7 +1,8 @@
-{ lib, mkDerivation, fetchFromGitHub
+{ lib, mkDerivation, fetchFromGitHub, stdenv
```

```suggestion
{ stdenv, mkDerivation, fetchFromGitHub
```

Also update meta with to stdenv.lib.